lib/mavenlink/client.rb

```diff
@@ -87,16 +91,16 @@ def connection_options
       }.freeze
     end
 
-    def parse_request(response)
-      if response.present?
-        parsed_response = JSON.parse(response)
+    def parse_request(body, status)
```

Comment:
what if response doesn't have a status?

Reply:
A response is almost certainly guaranteed to have a `.status` if it has a `.body`